api/incubator/src/main/java/io/opentelemetry/api/incubator/config/internal/YamlStructuredConfigProperties.java

```diff
@@ -85,13 +88,17 @@ static YamlStructuredConfigProperties create(Map<String, Object> properties) {
         mapEntries.put(key, configProperties);
         continue;
       }
-      throw new ConfigurationException(
+      if (value == null) {
+        nullKeys.add(key);
+        continue;
+      }
+      throw new StructuredConfigException(
```

Comment:
should StructuredConfigException accept a type param to normalize the message a bit? not required to action this